src/languages/en.json

```diff
@@ -42,7 +65,46 @@
         }
       }
     },
+    "SoicalAndLegal": {
+      "docs": "Docs",
+      "terms-of-use": "Terms of Use",
+      "risk-factor": "Risk Factor"
+    },
+    "NavLinks": {
+      "vaults": "Vaults",
+      "gov": "Gov",
+      "airdrop-machine": "Airdrop Machine",
+      "submit-vulnerability": "Submit Vulnerability",
+      "committee-tools": "Committee Tools",
+      "vault-editor": "Vault Editor"
+    },
     "DepositWithdraw": {
+      "deposit": "DEPOSIT",
+      "withdraw": "WITHDRAW",
+      "max": "MAX",
+      "claim": "CLAIM",
+      "and-claim": "AND CLAIM",
+      "text-0": "Vault token",
+      "text-1": "Insufficient funds",
+      "text-2": "Can't withdraw more than available",
+      "text-3": "I UNDERSTAND AND AGREE TO THE",
+      "text-4": "TERMS OF USE",
+      "text-5": "COMMITTEE IS NOT CHECKED IN YET!",
+      "text-6": "DEPOSIT PAUSE IS IN EFFECT!",
+      "text-7": "SAFE PERIOD IS ON. WITHDRAWAL IS NOT AVAILABLE DURING SAFE PERIOD",
+      "text-8": "DEPOSIT WILL CANCEL THE WITHDRAWAL REQUEST",
+      "text-9": "WITHDRAWAL REQUEST",
+      "text-10": "CHECK IN",
+      "text-11": "Balance:",
+      "text-12": "Balance to withdraw:",
```

Comment:
nice work Roie :) I think is better if we have 'legible' keys. For example I would change:

from -> "text-5": "COMMITTEE IS NOT CHECKED IN YET!",
to -> "committeeNotCheckedIn": "COMMITTEE IS NOT CHECKED IN YET!",

from -> "text-7": "SAFE PERIOD IS ON. WITHDRAWAL IS NOT AVAILABLE DURING SAFE PERIOD",
to -> "safePeriodIsOn": "SAFE PERIOD IS ON. WITHDRAWAL IS NOT AVAILABLE DURING SAFE PERIOD",

It's better to know where are you in the code, it would be easier :D

Reply:
yes, only put something short that represents the whole text.

for example: 
"safePeriodIsOn": "SAFE PERIOD IS ON. WITHDRAWAL IS NOT AVAILABLE DURING SAFE PERIOD",

Reply:
I'm more talking about long text like a paragraph @fonstack

Reply:
For example FAQ section in the airdrop. There I did like `question-1` and `answer-1` @fonstack

Reply:
for that example I would keep `question-1` and `answer-1`. It's ok :)